### DISCUSSION TITLE
Support Alcohol 120% MDF/MDS disc image format

### DISCUSSION THREAD

**Discussion by Winins** [86Box/86Box#4256] [Ideas]
As MDF and KF(KryoFlux) are the main CD and floppy formats, respectively, on BetaArchive, and since most releases are that way, I think there should be support for these formats.

**Comment by dhrdlicka** [comment]
The Kryoflux part is a dupe of #291.

**Comment by sskras** [comment]
Does the content of optical disk images have to be accessed as a files from the mounted ISO9660 file system? 

If yes, maybe mounting them via cdemu / wincdemu would do?  

If no, is Alcohol 120% images something that cannot be decoded by the libmirage into some common image format:
https://cdemu.sourceforge.io/about/libmirage ? 

The latter being C-only library which eg. isn't being loved by at least one C++ developer: https://sysprogs.com/w/forums/topic/new-features-request/#post-1005

**Comment by Winins** [comment]
The files cannot be accessed on a regular Windows environment, and libmirage supports it.

**Reply by sskras** [reply]
Thanks for clarification. Hence my suggestion to rely on `libmirage` instead of the own implementation. :)

**Reply by OBattler** [reply]
You're replying to a post from 2019, when we did not support MDS yet.

**Comment by mishamosher** [comment]
A list of several open source MDS/MDF software/code, in case it helps understanding the spec:
- https://github.com/TASVideos/BizHawk/pull/1034
- https://github.com/rzippo/MDFtoISO
- https://github.com/aaru-dps/Aaru/tree/master/Aaru.Images/Alcohol120
- https://github.com/sysprogs/WinCDEmu/tree/master/ImageFormats/MDS
- https://github.com/sizious/img4dc/tree/master/mds4dc

**Comment by ucucanok** [comment]
Done as of build 7274
